### PR TITLE
Create_bindings() gains type argument

### PR DESCRIPTION
**Pull request by krlmlr**
for loading bindings without packages.

Includes cosmetic changes -- ~~roxygen2 upgrade and~~ NAMESPACE fix.

### CODE REVIEW

**Comment by richfitz** [comment]
Ah. Are you using `create_bindings`? Because I have removed it in the refactor branch in an aim to simplify the packages and remove bits I don't believe are working well...

**Comment by krlmlr** [comment]
It's helpful to have the bindings available in a fresh session, and I'm restarting R fairly often. Will there be a replacement?

**Comment by richfitz** [comment]
The closest to a replacement is `remake::dump_environment()`.  But if there's a use case I can undo the removal (part of the issue with the current implementation was some hard-to-nail-down bugs with the static analysis that remake does to build the graph)

**Comment by krlmlr** [comment]
Actually, I quite like create_bindings() for its laziness.

**Comment by fmichonneau** [comment]
For what it's worth, I also rely heavily on `create_bindings()`

**Comment by richfitz** [comment]
Thanks for letting me know @fmichonneau!  I'll leave them in then :)

**Comment by krlmlr** [comment]
Now also temporarily changing directory when evaluating an active binding, a step towards #107.

File: R/remake.R [hunk 1]
```diff
@@ -1,4 +1,5 @@
-#' @importFrom utils packageName packageVersion
+#' @importFrom utils packageName packageVersion browseURL download.file install.packages unzip zip
+#' @importFrom grDevices col2rgb dev.cur dev.off rgb
```
Comment (richfitz) [inline R/remake.R]: It's all good - in other packages I just `@import` the whole thing :)

**Comment by krlmlr** [comment]
I'm not sure about the type argument anymore, but the second commit 4acaba2 is ready for merging, I think. Mind if I cherry-pick it?

**Comment by richfitz** [comment]
Yeah, please do.

Apologies for the continued lack of progress on my part - have picked up a second cold to pair with the first that just finished, so have been sub par for too long :(

**Comment by krlmlr** [comment]
No worries. Get well!

**Comment by krlmlr** [comment]
Do we also need a type argument for delete_bindings()?

**Comment by richfitz** [comment]
Yes sounds good. Do you want to add it in here or a new PR (and in which case I'll merge this)

**Comment by krlmlr** [comment]
I think this should be added here, if at all.

**Comment by krlmlr** [comment]
Yeah, that's the key word. The original use case has vanished by now, I'd rather like to specify which targets to create bindings for (and always load packages/helpers, which should be reasonably fast after they have been loaded once).

**Pull request closed without merging**